packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -90,6 +90,8 @@ contract StableToken is
    * @param registryAddress Address of the Registry contract.
    * @param inflationRate weekly inflation rate.
    * @param inflationFactorUpdatePeriod how often the inflation factor is updated.
+   * @param initialBalanceAddresses array of addresses with an initial balance.
+   * @param initialBalanceValues array of balance values corresponding to initialBalanceAddresses.
```

Comment:
Nit: looks like there's some leftover inconsistency (in this natspec and potentially throughout the file) - have param descriptions start with capital letter, end with period.